lib/ex_machina/ecto.ex

```diff
@@ -85,7 +94,7 @@ defmodule ExMachina.Ecto do
   @doc """
   Saves a record using `Repo.insert!` when `create` is called.
   """
-  def save_record(module, repo, record) do
+  def save_record(_module, repo, record) do
```

Comment:
This makes me wonder if we should even pass the module to save_record at all. I'll change that in a separate PR